src/components/UI/Display/Cluster/ClusterIDLabel/index.tsx

```diff
@@ -42,14 +42,21 @@ const Label = styled.span<{ clusterID: string }>`
   border-radius: 0.2em;
 `;
 
+export enum ClusterIDLabelType {
+  Name = 'name',
+  ID = 'ID',
+}
+
 interface IClusterIDLabelProps extends React.ComponentPropsWithoutRef<'span'> {
   clusterID: string;
   copyEnabled?: boolean;
+  variant?: ClusterIDLabelType;
 }
 
 const ClusterIDLabel: React.FC<IClusterIDLabelProps> = ({
   clusterID,
   copyEnabled,
+  variant = ClusterIDLabelType.ID,
```

Comment:
Throughout the codebase we use a component's `defaultProps` static property to set default values for props. [They are then handled internally by `react`](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values), and behave in the same way.

https://github.com/giantswarm/happa/blob/d8f609f8bd5a8d96797f46413811309bb49065d5/src/components/UI/Display/MAPI/clusters/ClusterDetail/ClusterDetailWidget/index.tsx#L64

The way you implemented it is cleaner and a bit closer to 2021, but for consistency, it would be great if we could use `defaultProps` here, as well.

If you think it's valuable, we can switch our codebase to using function argument defaults. It would definitely be more enjoyable to use with TypeScript.

Reply:
Let's stick with `defaultProps` for now for consistency's sake. I'll make the appropriate changes 👍